config/config.go

```diff
@@ -133,6 +133,9 @@ type Config struct {
 	// Environment override to return when resolving the current environment.
 	DatabricksEnvironment *environment.DatabricksEnvironment
 
+	// OIDC and WIF audience for the token
```

Comment:
This is a bit on the short side. Something like "When using Workload Identity Federation, the audience to specify when fetching an ID token from the ID token supplier." would be a bit better.